nais/dev/nais.yaml

```diff
@@ -35,4 +35,4 @@ spec:
     - name: DPI_DP_GRAPHQL
       value: http://dp-graphql
     - name: DPI_DP_REGEL_API
-      value: http://dp-regel-api
\ No newline at end of file
+      value: http://dp-regel-api.teamdagpenger
```

Comment:
gjør det lettere for oss selv dersom vi ikke trenger namspace i url. 

MAO slett dp-regel-api fra default namespace når vi har migrert appene som kall dp-regel-api til teamnamespace.

Reply:
ah, okei